### PR TITLE
Update scuttle-gamepad.md

### PR DESCRIPTION
**Pull request by mestcihazal**
updated following this conversation: https://discord.com/channels/1083489952408539288/1126875127972376627

### CODE REVIEW

File: docs/tutorials/control/scuttle-gamepad.md [hunk 1]
```diff
@@ -76,6 +62,20 @@ Services are the software packages that provide our robots with cool and powerfu
 
 <img src="../../img/scuttle-gamepad/pi-game-create-service.png" alt="Create service builder" />
 
+<li>The Viam app does not add any JSON attributes to the Base Remote Control configuration.
+
```
Comment (JessamyT) [inline docs/tutorials/control/scuttle-gamepad.md:66]:
```suggestion
```
Does this line need to be here?

Comment (JessamyT) [inline docs/tutorials/control/scuttle-gamepad.md:66]:
```suggestion
```
Does this line need to be here?

File: docs/tutorials/control/scuttle-gamepad.md [hunk 2]
```diff
@@ -76,6 +62,20 @@ Services are the software packages that provide our robots with cool and powerfu
...
+Please add the following Attributes:
```
Comment (JessamyT) [inline docs/tutorials/control/scuttle-gamepad.md:67]:
```suggestion
Copy and paste the following into the empty **Attributes** field:
```
[optional]

**Review by JessamyT — APPROVED** [review]
Couple of optional nits. Also looking this over I see this is generally pretty out of date but that's beyond this PR's scope! Thanks for fixing!

File: docs/tutorials/control/scuttle-gamepad.md (hunk 1 above)
Comment (mestcihazal) [reply docs/tutorials/control/scuttle-gamepad.md:66]: not really haha!

**Pull request merged**